Apps/frontend/src/views/Oscilloscope.svelte

```diff
@@ -31,6 +34,7 @@
 <div class="wrapper" data-cy="oscilloscope">
   <div class="stack coordinate-system">
     <CoordinateSystem yScale={scaleY} />
+    <OnOffButton on:switch-plot-enabled={(e) => {isEnabled = e.detail.enabled;}} />
```

Comment:
I'd prefer the OnOffButton outside of the div.wrapper to 1. prevent weird any absolute positioning bugs
2. have the controls separated from the representation